src/main/java/net/minecraftforge/event/world/DimensionPreloadEvent.java

```diff
@@ -0,0 +1,35 @@
+package net.minecraftforge.event.world;
+
+import net.minecraft.world.DimensionType;
+import net.minecraftforge.common.DimensionManager;
+import net.minecraftforge.fml.common.eventhandler.Cancelable;
+import net.minecraftforge.fml.common.eventhandler.Event;
+import net.minecraftforge.common.MinecraftForge;
+
+/**
+ * Created by goreacraft on 17/03/03.
```

Comment:
Just make a new commit. Github can squash commits when merging your PR, so you do not need to do that yourself.